src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -46,6 +46,14 @@ public AddCommand(String name,
         for (String tagName : tags) {
             tagSet.add(new Tag(tagName));
         }
+		
+		String[] nameSplit = name.split("\\s+");
```

Comment:
Just a minor comment. Please try to ensure your indentation is consistent for lines belonging to the same nesting level.